### PR TITLE
Np create test for multiome

### PR DESCRIPTION
**Pull request by nikellepetrillo**
### Description

This PR adds the test for the Multiome pipeline.
----

### Checklist 
If you can answer "yes" to the following items, please add a checkmark next to the appropriate checklist item(s) **and** notify our WARP documentation team by tagging either @ekiernan or @kayleemathews in a comment on this PR.

- [ ] Did you add inputs, outputs, or tasks to a workflow?
- [ ] Did you modify, delete or move: file paths, file names, input names, output names, or task names?
- [ ] If you made a changelog update, did you update the pipeline version number?

### CODE REVIEW

File: beta-pipelines/multiome/test_inputs/Scientific/10k_pbmc.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  ],
+  "Multiome.read2_fastq_gzipped":[
+    "gs://broad-gotc-test-storage/Multiome/input/scientific/10k_PBMC_Multiome/10k_PBMC_Multiome_nextgem_Chromium_Controller_atac_S1_L001_R2_001.fastq.gz",
+    "gs://broad-gotc-test-storage/Multiome/input/scientific/10k_PBMC_Multiome/10k_PBMC_Multiome_nextgem_Chromium_Controller_atac_S1_L002_R2_001.fastq.gz"
+  ],
+  "Multiome.read3_fastq_gzipped":[
+    "gs://broad-gotc-test-storage/Multiome/input/scientific/10k_PBMC_Multiome/10k_PBMC_Multiome_nextgem_Chromium_Controller_atac_S1_L001_R3_001.fastq.gz",
+    "gs://broad-gotc-test-storage/Multiome/input/scientific/10k_PBMC_Multiome/10k_PBMC_Multiome_nextgem_Chromium_Controller_atac_S1_L002_R3_001.fastq.gz"
+  ],
+  "Multiome.ref_genome_fasta":"gs://broad-gotc-test-storage/Multiome/input/scientific/10k_PBMC_Multiome/GRCh38.primary_assembly.genome.fa",
```
Comment (nikellepetrillo) [inline beta-pipelines/multiome/test_inputs/Scientific/10k_pbmc.json:30]: note to self, change the reference paths to GCP public bucket once they are available and clean up in broad-gotc-test-storage

File: tests/broad/scala_test/src/main/scala/org/broadinstitute/dsp/pipelines/commandline/PipelineTestType.scala [hunk 2]
```diff
@@ -91,6 +91,12 @@ object PipelineTestType extends Enum[PipelineTestType] {
         "JointGenotyping",
         "/broad/dna_seq/germline/joint_genotyping/"
       )
+  case object Multiome
```
Comment (nikellepetrillo) [inline tests/broad/scala_test/src/main/scala/org/broadinstitute/dsp/pipelines/commandline/PipelineTestType.scala:94]: will need to update the path once we move this out of beta-pipelines

File: tests/broad/scala_test/src/main/scala/org/broadinstitute/dsp/pipelines/tester/CromwellWorkflowTester.scala [hunk 3]
```diff
@@ -54,7 +54,7 @@ object CromwellWorkflowTester {
 
   lazy val GotcRoot: File = WarpRoot / "genomes_in_the_cloud"
 
-  lazy val PipelineRoot: File = WarpRoot / "pipelines"
+  lazy val PipelineRoot: File = WarpRoot / "beta-pipelines"
```
Comment (nikellepetrillo) [inline tests/broad/scala_test/src/main/scala/org/broadinstitute/dsp/pipelines/tester/CromwellWorkflowTester.scala:57]: note to self, need to change this back to "pipelines" as the root

**Comment by nikellepetrillo** [comment]
Note to self, currently running updating truth here: https://cromwell.gotc-dev.broadinstitute.org/api/workflows/1/61f64f3b-1636-4299-90af-5b9a9719503c/status so i can test out the test

**Comment by nikellepetrillo** [comment]
successful test: https://cromwell.gotc-dev.broadinstitute.org/api/workflows/1/af9c3db7-e733-488e-8ac0-6065b3b122bd/metadata

File: pipelines/skylab/multiome/atac.json [hunk 4]
```diff
@@ -4,7 +4,7 @@
   "ATAC.TrimAdapters.adapter_seq_read1": "GTCTCGTGGGCTCGGAGATGTGTATAAGAGACAG",
   "ATAC.TrimAdapters.adapter_seq_read2": "TCGTCGGCAGCGTCAGATGTGTATAAGAGACAG",
   "ATAC.output_base_name": "scATAC",
-  "ATAC.monitoring_script": "gs://fc-51792410-8543-49ba-ad3f-9e274900879f/cromwell_monitoring_script2.sh",
+  //"ATAC.monitoring_script": "gs://fc-51792410-8543-49ba-ad3f-9e274900879f/cromwell_monitoring_script2.sh",
```
Comment (ekiernan) [inline pipelines/skylab/multiome/atac.json:7]: Why are the slashes needed here and not other inputs?

File: pipelines/skylab/multiome/multiome.wdl [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+
+import "../../../pipelines/skylab/multiome/atac.wdl" as atac
+import "../../../pipelines/skylab/optimus/Optimus.wdl" as optimus
+
+workflow Multiome {
+    String pipeline_version = "1.0.0"
+
+    input {
+        # Optimus Inputs
+        String counting_mode = "sn_rna"
```
Comment (ekiernan) [inline pipelines/skylab/multiome/multiome.wdl:11]: @nikellepetrillo I think the WDL might not match what's currently on develop in beta-pipelines. Aseel made changes to clean up the input names so there was no confusion about which fastqs were ATAC vs. gex. https://github.com/broadinstitute/warp/blob/develop/beta-pipelines/multiome/multiome.wdl

File: pipelines/skylab/multiome/test_inputs/Scientific/10k_pbmc.json [hunk 6]
```diff
@@ -0,0 +1,37 @@
+{
+"Multiome.annotations_gtf":"gs://broad-gotc-test-storage/Multiome/input/scientific/10k_PBMC_Multiome/introns_modified_gencode.vGencode_V43.primary_assembly.annotation.gtf",
+  "Multiome.barcodes_in_read_name":"true",
+  "Multiome.i1_fastq":[
```
Comment (ekiernan) [inline pipelines/skylab/multiome/test_inputs/Scientific/10k_pbmc.json:4]: The JSON will need to be updated to match the latest WDL in beta-pipeline folder.

File: verification/test-wdls/TestMultiome.wdl [hunk 7]
```diff
@@ -0,0 +1,212 @@
+version 1.0
+
+
+import "../../pipelines/skylab/multiome/multiome.wdl" as Multiome
+import "../../verification/VerifyMultiome.wdl" as VerifyMultiome
+import "../../tasks/broad/Utilities.wdl" as Utilities
+import "../../tasks/broad/CopyFilesFromCloudToCloud.wdl" as Copy
+
+workflow TestMultiome {
```
Comment (ekiernan) [inline verification/test-wdls/TestMultiome.wdl:9]: The inputs will also need to be updated to match the latest changes in beta-pipelines folder.

File: pipelines/skylab/multiome/multiome.wdl (hunk 5 above)
Comment (nikellepetrillo) [reply pipelines/skylab/multiome/multiome.wdl:11]: thanks for catching that @ekiernan 👍  should be all updated now

File: pipelines/skylab/multiome/atac.json (hunk 4 above)
Comment (nikellepetrillo) [reply pipelines/skylab/multiome/atac.json]: commented it out for testing but it is now uncommented 👍

File: verification/test-wdls/TestMultiome.wdl [hunk 8]
```diff
@@ -0,0 +1,213 @@
...
+                                    
+    ])
+
+    
+    # Collect all of the pipeline metrics into single Array[String]
+    Array[String] pipeline_metrics = flatten([
+                                    [ # File outputs
+                                    Multiome.gene_metrics,
+                                    Multiome.cell_metrics,
+                                    ],
```
Comment (khajoue2) [inline verification/test-wdls/TestMultiome.wdl:125]: The spacing looks weird here. Not sure if it is my browser…

**Review by khajoue2 — APPROVED** [review]

File: verification/test-wdls/TestMultiome.wdl (hunk 8 above)
Comment (khajoue2) [reply verification/test-wdls/TestMultiome.wdl:125]: The last value in the array is separated by a comma and makes it of length 3.

File: pipelines/skylab/multiome/atac.json [hunk 9]
```diff
@@ -7,4 +7,4 @@
   "ATAC.monitoring_script": "gs://fc-51792410-8543-49ba-ad3f-9e274900879f/cromwell_monitoring_script2.sh",
   "ATAC.tar_bwa_reference": "gs://broad-gotc-test-storage/scATAC/input/plumbing/reference/mm10/mm10.tar",
   "ATAC.barcodes_in_read_name": "true"
```
Comment (ekiernan) [inline pipelines/skylab/multiome/atac.json:9]:
```suggestion
  "ATAC.barcodes_in_read_name": "false"

File: pipelines/skylab/multiome/test_inputs/Scientific/10k_pbmc.json [hunk 10]
```diff
@@ -0,0 +1,36 @@
+{
+"Multiome.annotations_gtf":"gs://broad-gotc-test-storage/Multiome/input/scientific/10k_PBMC_Multiome/introns_modified_gencode.vGencode_V43.primary_assembly.annotation.gtf",
+  "Multiome.barcodes_in_read_name":"true",
```
Comment (ekiernan) [inline pipelines/skylab/multiome/test_inputs/Scientific/10k_pbmc.json:3]:
```suggestion
  "Multiome.barcodes_in_read_name":"false",

**Review by ekiernan — APPROVED** [review]
LGTM! Thanks, Nikelle. Just added a suggestion to make the barcodes_in_readname false because we have them in the CB tag by default now.

**Pull request merged**